src/main/kotlin/com/liftric/dtcp/tasks/GenerateVex.kt

```diff
@@ -82,8 +82,8 @@ abstract class GenerateVexTask : DefaultTask() {
         vexFile.metadata.component = sbom.metadata.component ?: Component()
         val pluginData = Tool().apply {
             vendor = props.getProperty("vendor", "Liftric")
-            name = props.getProperty("name", "Liftric")
-            version = props.getProperty("version", "Liftric")
+            name = props.getProperty("name", "dependency-track-companion-plugin")
```

Comment:
I think it would be better to use `public String getProperty(String key)` instead and throw an error if it is null. Better than wondering why the wrong data is used, maybe for countless vex files until somebody notices it.